pkg/cloudprovider/providers/openstack/openstack_volumes.go

```diff
@@ -578,6 +592,9 @@ func (os *OpenStack) GetAttachmentDiskPath(instanceID, volumeID string) (string,
 
 // DiskIsAttached queries if a volume is attached to a compute instance
 func (os *OpenStack) DiskIsAttached(instanceID, volumeID string) (bool, error) {
+	if instanceID == "" {
+		glog.Warningf("calling DiskIsAttached with empty instanceid: %s %s", instanceID, volumeID)
+	}
```

Comment:
well there was case where the code did not work properly if instanceid was "". I can remove it, but I was thinking that it could help debugging in future.